todo-app/src/List-Items.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import TodoItems from "./Todos";
+
+export default class ListItems extends Component {
```

Comment:
What exactly is the purpose of this `ListItems` component? Does it accomplish anything that cannot be accomplished by only having a `Todos` component instead of having both a `ListItems` _and_ a `Todos` component?

Also just a small note, it's convention to have your filename match the component name exactly. Your component name is `ListItems`, so your file should be named `ListItems.js`.